README.md

```diff
@@ -2346,6 +2346,28 @@ remains the same throughout the file.  This is a common convention
 followed by most Clojure source code, and required by several other
 Clojure development tools.
 
+## Ignored faults
+
+If there are specific instances of linter faults that you need to supress
+(e.g. for making a CI build pass), you can use the `:ignored-faults` option.
+
+It has the following shape:
+
+```clj
+;;linter-name            ns-name            target
+;;---                    ---                ---
+{:implicit-dependencies {'example.namespace {:line 3 :column 2}} 
+ :unused-ret-vals       {'another.namespace true}}
+```
+
+`{:line 3 :column 2}` matches a given linter (within a specific ns)
```

Comment:
Would it be possible to allow for only line-numbers?

Reply:
> Would it be possible to allow for only line-numbers?

Good idea! Yes it should be possible to have that middle ground.

Will impl

Reply:
Done. It also made me realise that this API should expect vectors of maps (vs. discretes maps), since a given file can have _multiple_ instances of a linter to silence.

If a user happens to pass a single hashmap, we simply wrap it into a vector, as a lenient API can be considered more friendly.